etc/config.xml

```diff
@@ -55,5 +55,77 @@
                 <stockprice>0 8 * * *</stockprice>
             </cron>
         </sinchimport>
+        <csp>
+            <mode>
+                <storefront>
+                    <report_only>1</report_only>
```

Comment:
Setting config to enforce report-only actually constitutes a downgrade in security on Magento 2.4.7+. This is because in 2.4.6 and lower, the mode is report-only by default (thus this doesn't change anything and isn't needed), and in 2.4.7+, the mode is restrict for all payment pages (e.g. checkout, where this change prevents enforcement, and thus shouldn't be included).
Please remove